app/build.gradle

```diff
@@ -1,43 +1,54 @@
 apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
 
     defaultConfig {
-        applicationId "com.instacart.library.truetime"
+        applicationId "com.instacart.truetime"
         minSdkVersion 14
         targetSdkVersion rootProject.ext.targetSdkVersion
 
         versionCode rootProject.ext.versionCode
         versionName rootProject.ext.versionName
 
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
     buildTypes {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    buildFeatures {
+        viewBinding true
+    }
 }
 
 repositories {
     maven {
         url "https://jitpack.io"
     }
+    mavenCentral()
 }
 
-ext {
-    butterknifeVersion = '8.8.1'
-}
 dependencies {
     implementation project(path: ':library')
-    implementation project(path: ':library-extension-rx')
-//    compile 'com.github.instacart.truetime-android:library-extension-rx:master-SNAPSHOT'
-
-    implementation 'com.android.support:appcompat-v7:27.1.1'
-    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
-    implementation "com.jakewharton:butterknife:$butterknifeVersion"
-    annotationProcessor "com.jakewharton:butterknife-compiler:$butterknifeVersion"
-    implementation "io.reactivex.rxjava2:rxandroid:2.0.2"
+//    implementation project(path: ':library-extension-rx')
+    implementation 'com.github.instacart.truetime-android:library-extension-rx:3.5'
+
+    implementation "org.jetbrains.kotlin:kotlin-stdlib"
```

Comment:
This shouldn't be needed, it gets added by default when you apply the kotlin plugin

Reply:
🤔 AS is complaining when i remove this. I'm guessing i've stripped/messed with the dependencies during the kotlin migration. i'm going to leave this one as is. once we have it merged, happy to try again and clean this up.